### ISSUE TITLE
File exists error

### ISSUE THREAD

**Issue by Seme4eg** [vinceliuice/Elegant-grub2-themes#6]
Trying to install on fresh pc, and the script reports an error right away:

`cp: cannot create regular file '/boot/grub/themes/Elegant-fore..../icons/opensuse.png': File exists`

Ofcourse i cleaned this dir and the issue still persists.

**Comment by vinceliuice** [comment]
What command did you run, I did not find this issue

**Comment by Seme4eg** [comment]
sudo ./install.sh -s 2k -b

**Comment by vinceliuice** [comment]
What's your system version ?

**Comment by Seme4eg** [comment]
Arch Linux x86_64 
Kernel: 6.11.6-arch1-1

**Comment by Paspartame** [comment]
Is your `/boot` partition formatted as FAT32? The target folder `icons` will contain both `opensuse.png` and `openSUSE.png`, which is not allowed on a FAT32 file system.

**Comment by Seme4eg** [comment]
yes its fat32 as i was installing arch based on instructions from arch wiki. 
On my previous laptop it was also fat32, i installed elegant theme successfully. Reason i can't install it now is [that](https://github.com/vinceliuice/Elegant-grub2-themes/commit/327118f1d4f5d6419f78e72364f8f752966543f3) commit.
Because i hard resetted to the commit before that one and installed theme successfully.

**Comment by AtelierMizumi** [comment]
I can confirm after revert back to [327118f
Preview](https://github.com/vinceliuice/Elegant-grub2-themes/commit/327118f1d4f5d6419f78e72364f8f752966543f3)  the script worked perfectly

Temporary fix:

After cloning revert the state of local repo to 327118f

```bash
git reset --hard 327118f
```

Use install script normally

**Comment by abhijithabhiakl** [comment]
Is this issue solved ? Im facing the same issue on arch linux kernel 6.12.34-1-lts

**Comment by abhijithabhiakl** [comment]
> I can confirm after revert back to [327118f Preview](https://github.com/vinceliuice/Elegant-grub2-themes/commit/327118f1d4f5d6419f78e72364f8f752966543f3) the script worked perfectly
> 
> Temporary fix:
> 
> After cloning revert the state of local repo to [327118f](https://github.com/vinceliuice/Elegant-grub2-themes/commit/327118f1d4f5d6419f78e72364f8f752966543f3)
> 
> git reset --hard 327118f
> 
> Use install script normally

this fixed
